region/norfair/east/Frog Speedway.json

```diff
@@ -842,7 +842,7 @@
   "notables": [
     {
       "id": 1,
-      "name": "Shot Block Overload (Speedless Speedway)",
+      "name": "Shot Block Overload",
```

Comment:
I like "Speedless Speedway" more

Reply:
Ok, reverted this room. "Speedless Speedway" doesn't seem as descriptive of what is happening in these strats, compared to "Shot Block Overload", but at least it conveys that it's done without Speedbooster, which is useful. The room name isn't too long here, so we can just leave both descriptors without much issue.